pkgs/development/interpreters/python/python-packages-base.nix

```diff
@@ -40,21 +40,27 @@ let
         result
     );
 
+  allowFinalAttrs = f: lib.mirrorFunctionArgs f (args: f (lib.fix (lib.toFunction args)));
```

Comment:
Isn't there a lib function that does that?

If the process could be standardized that could be done to all builders and we could do a rec witch hunt

Reply:
That's https://github.com/NixOS/nixpkgs/pull/234651 (for python https://github.com/NixOS/nixpkgs/pull/271387), but it is stale and unlikely to be merged soon. This PR implements a temporary solution